msteams-platform/concepts/app-fundamentals-overview.md

```diff
@@ -1,10 +1,41 @@
 ---
-title: Teams platform development fundamentals
+title: Teams platform development fundamentals overview
 author: heath-hamilton
 description: Describe the foundational concepts of Teams platform development.
 ms.topic: conceptual
 ms.author: lajanuar
 ---
+
 # Teams platform development fundamentals
 
-Coming soon ...
+Microsoft Teams app fundamentals give the direction you need to create your custom Teams app. You can recognize the framework required to plan your Teams app. The document helps you to understand the user-app communication and figure out the kind of app surfaces you need to use or the APIs your app might require in the process. Get some inspiration to embrace interactivity that can deepen the app experience when you integrate with Teams.
+
+## Capabilities overview and entry points
+
+This article talks about the multiple ways to extend your Teams app. It gives you needed guidance on all the core capabilities and the entry points that work in a collaborative space. You can experiment with the extension points for building your apps. Important app project components help you to correctly configure your app page. Teams app can have [multiple capabilities](../concepts/capabilities-overview.md) and [entry points](../concepts/extensibility-points.md). The information about all the core capabilities is given in this article for you to familiarize yourself with Teams app development.
```

Comment:
@KirtiPereira In general, we should avoid saying "this article" or "this document". Also, it is not clear this text is referring to another article. Someone may interpret it as meaning the page they're already on.